### ISSUE TITLE
EMC values changing

### ISSUE THREAD

**Issue by olatnip** [sinkillerj/ProjectE#467]
The EMC values of  the Anti-Matter Relay MK3, Red Matter Furnace, and Volcanite Amulet were suddenly changed to 7,408, 2,094, and 13 respectfully.

**Comment by MaPePeR** [comment]
Are you using the latest version of the Mod? I hoped that this bug was only present in old versions :/

**Comment by olatnip** [comment]
Yes I am sadly

**Comment by sinkillerj** [comment]
Could you elaborate on when these values were changed? Without steps to reproduce its going to be hard to find the source of this.

**Comment by olatnip** [comment]
Well I'm not actually sure what caused the issue. I was just played projecte on the direwolf20 pack, when suddenly I noticed that I was able to produce more of the relays than I should have. After that I checked the other projecte items and found the other two had lowered EMC as well. Sorry if that is not enough information to reproduce the glitch, from my perspective it didn't seem to be directly caused from anything in particular. Hopefully if anyone else has experienced this issue they can provide more details.
